hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/hip869/NodeCreateTest.java

```diff
@@ -263,13 +270,12 @@ final Stream<DynamicTest> failOnTooManyServiceEndpoints() {
     final Stream<DynamicTest> allFieldsSetHappyCaseForDomains() {
         return hapiTest(
                 newKeyNamed(ED_25519_KEY).shape(KeyShape.ED25519),
-                overriding("nodes.gossipFqdnRestricted", "false"),
                 nodeCreate("nodeCreate")
                         .description("hello")
                         .gossipCaCertificate("gossip".getBytes())
                         .grpcCertificateHash("hash".getBytes())
                         .accountId(asAccount("0.0.100"))
-                        .gossipEndpoint(GOSSIP_ENDPOINTS)
+                        .gossipEndpoint(GOSSIP_ENDPOINTS_IPS)
```

Comment:
Why did you change this? With this change, the test becomes identical to allFieldsSetHappyCaseForIps() case defined below.

Reply:
They not the same. I wanted to test the default nodes.gossipFqdnRestricted setting without overriding, gossipEndpoint can only accept ip address, not domain name. You had nodes.gossipFqdnRestricted overrides in both allFieldsSetHappyCaseForDomains and allFieldsSetHappyCaseForIps.

Reply:
Ok, but now there is no place that domain names are being tested. There needs to be a test that overrides gossipFqdnRestricted and then uses domain names.

Reply:
Updated the test with domain names.

Reply:
Looks good, thanks!